addons/udes_stock/models/stock_move_line.py

```diff
@@ -230,3 +230,14 @@ def _split(self, uom_qty=None):
             )
             res = new_ml
         return res
+
+    def get_lines_todo(self):
```

Comment:
Is this the same as `get_lines_incomplete`?

Reply:
Yes replacing :)